packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -0,0 +1,292 @@
+import * as React from 'react';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
+import { ApiReferencesTable, MEDIUM_GAP_SIZE, LARGE_GAP_SIZE } from './ApiReferencesTable';
+import {
+  IApiProperty,
+  IApiInterfaceProperty,
+  IApiEnumProperty,
+  IMethod,
+  IApiReferencesTableSetProps,
+  IEnumTableRowJson,
+  ITableRowJson
+} from './ApiReferencesTableSet.types';
+import { PropertyType } from '../../utilities/parser/index';
+
+export interface IApiReferencesTableSetState {
+  properties: Array<IApiProperty>;
+  showSeeMore: boolean;
+}
+
+const apiReferencesTableTopMargin = mergeStyles({
+  marginTop: '40px'
+});
+
+export class ApiReferencesTableSet extends React.Component<IApiReferencesTableSetProps, IApiReferencesTableSetState> {
+  constructor(props: IApiReferencesTableSetProps) {
+    super(props);
+
+    this._onClickSeeMore = this._onClickSeeMore.bind(this);
```

Comment:
Please use a lambda function instead of bind: `private _onClickSeeMore = (): void => { ... }`